src/test/java/org/openrewrite/java/testing/jmockit/JMockitMockUpToMockitoTest.java

```diff
@@ -0,0 +1,618 @@
+              import org.junit.Test;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      new MockUp<MockUpClass>() {
+                          @Mock
+                          public void changeMsg() {
+                              MockUpClass.Save.msg = "mockMsg";
+                          }
+              
+                          @Mock
+                          public void changeText(String text) {
+                              MockUpClass.Save.text = "mockText";
+                          }
+                      };
+              
+                      assertEquals("mockMsg", new MockUpClass().getMsg());
+                      assertEquals("mockText", new MockUpClass().getText());
+                  }
+              
+                  public static class MockUpClass {
+                      public static class Save {
+                          public static String msg = "msg";
+                          public static String text = "text";
+                      }
+              
+                      public final String getMsg() {
+                          changeMsg();
+                          return Save.msg;
+                      }
+              
+                      public void changeMsg() {
+                          Save.msg = "newMsg";
+                      }
+              
+                      public String getText() {
+                          changeText("newText");
+                          return Save.text;
+                      }
+              
+                      public static void changeText(String text) {
+                          Save.text = text;
+                      }
+                  }
+              }
+              """,
+            """
+              import static org.junit.Assert.assertEquals;
+              import static org.mockito.AdditionalAnswers.delegatesTo;
+              import static org.mockito.ArgumentMatchers.*;
+              import static org.mockito.Mockito.*;
+              
+              import org.junit.Test;
+              import org.mockito.MockedConstruction;
+              import org.mockito.MockedStatic;
+              
+              public class MockUpTest {
+                  @Test
+                  public void test() {
+                      MockUpTest.MockUpClass mockObjMockUpTest_MockUpClass = mock(MockUpTest.MockUpClass.class, withSettings().defaultAnswer(CALLS_REAL_METHODS));
+                      doAnswer(invocation -> {
+                          MockUpClass.Save.msg = "mockMsg";
+                          return null;
+                      }).when(mockObjMockUpTest_MockUpClass).changeMsg();
+                      try (MockedStatic mockStaticMockUpTest_MockUpClass = mockStatic(MockUpTest.MockUpClass.class);MockedConstruction mockConsMockUpTest_MockUpClass = mockConstructionWithAnswer(MockUpTest.MockUpClass.class, delegatesTo(mockObjMockUpTest_MockUpClass))) {
```

Comment:
If possible, could you move the MockedConstruction to the next line, would be easier to read. Tempted to use a try finally instead here for readability, but it has it's downsides.

Reply:
I think the maybeAutoFormat() doesn't cover this case. I tried adding newline manually, but it didn't add proper indentations.

BTW, I tried https://github.com/openrewrite/rewrite-testing-frameworks/pull/601 and it output result like this
<img width="1365" alt="image" src="https://github.com/user-attachments/assets/54511982-0f0e-486d-b24d-f1ac347ac62a">

@timtebeek shall we fix this?

Reply:
The actual with nested try still looks better I guess than expected. The variable names also look good here.